EkaterinaUlitina/src/main/java/HomeWorkFromLesson3/FactorialFor.java

```diff
@@ -1,7 +1,18 @@
 package HomeWorkFromLesson3;
 
+import java.util.Scanner;
+
 public class FactorialFor {
     public static void main(String[] args) {
-        System.out.println("Will write it later");
```

Comment:
Answer me something ;)

Reply:
It was just stub in the code to do it later ;)